README.md

```diff
@@ -1,30 +1,45 @@
-publication, must be reproducible by other researchers in the future. To this end,
-it is necessary to provide metadata on the exact versions used for all the software
-involved. Such a "snapshot" of the workflow should be published side-by-side with
-scientific papers.
-
-In this documentation, we want to evaluate if, or which, currently existing workflow
-tools can provide the means to accomplish the above-mentioned challenges, that is:
-
-- comprehensive documentation of the individual processing steps of a workflow,
-  their interaction and the flow of data
-- version-controlling the workflow and publishing it side-by-side with a scientific publication
-- packaging of the workflow together with the involved software packages to make it
-  shareable with and executable by other researchers
-
-The aim of this repository is to collect, in a community effort, the requirements
-of scientific workflows on such tools, and to document user experiences on how to
-set up workflows using existing tools.
-
-For a more detailed overview, see the [documentation](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/).
+of various pieces of software executed in a particular order. The output of a
+computational step may serve as input to a subsequent computation, which requires
+them to be processed sequentially with a proper mapping of outputs to inputs.
+Other computations are independent of each other and can be executed in parallel.
+Thus, one of the main tasks of a workflow tool is a proper and efficient scheduling
+of the individual processing steps.
+
+<center><img src="./img/workflows.png" width="400"></center>
+
+Each processing step, just as the workflow itself, typically processes some input and
+produces output data. Apart from changing the input data to operate on, processing
+steps can usually be configured by a set of parameters to change their behavior.
+Moreover, the behavior of a processing step is determined by its source code
+and/or executable binaries/packages that are called within it. Beyond this, the
+computation environment not only has a significant influence on its behavior, but
+is also crucial in order for the processing step to work at all. The environment
+includes the versions of the interpreters or compilers, as well as all third-party
+libraries and packages that contribute to the computations carried out in a
+processing step.
+
+In order for research workflows to comply with the
+[FAIR principles](https://www.go-fair.org/fair-principles/), they should be:
+
+- published
+- documented
+- annotated with metadata
+- reproducible
+- reusable
+
+Because of the dependency on their environment, reproducibility and reusability
+is not trivial to achieve for scientific workflows. The results presented in a
+scientific paper, for instance, are produced with a very specific environment,
+and in order to guarantee reproducibility, this environment has to be reinstantiated
+possibly many years later.
```

Comment:
IMHO this could be more detailed, explaining what exactly makes computational research reproducible (see this [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003285)) But I am not sure if the introduction is the right place. Do you think we should add a section on Reproducibility?